LenovoLegionToolkit.WPF/Controls/Automation/Steps/RunAutomationStepControl.cs

```diff
@@ -24,6 +24,18 @@ public class RunAutomationStepControl : AbstractAutomationStepControl<RunAutomat
         Width = 300,
     };
 
+    private readonly CheckBox _checkBoxProcessRunSilently = new()
+    {
+        Content = Resource.RunAutomationStepControl_ProcessRunSilently,
+        IsChecked = true,
```

Comment:
Assigning default values here shouldn't be needed - these checkboxes will be initialized in lines 104-105.